components/p3a/p3a_metrics.h

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+#define BRAVE_COMPONENTS_P3A_P3A_METRICS_H_
+
+namespace brave::p3a {
+
+// Allowlist for histograms that we collect.
+// A metric must be listed here to be reported.
+//
+// TODO(iefremov): Provide moar histograms!
+// Could be replaced with something built dynamically,
+// although so far we've found it a useful review point.
+//
+// clang-format off
```

Comment:
what is it for?

Reply:
clang-format re-line-wraps the list entries so there's more than one metric on some lines. That makes diffs harder to read.

This isn't new; we have it in the current version too.